sherpa/astro/io/pyfits_backend.py

```diff
@@ -728,33 +720,80 @@ def get_rmf_data(arg, make_copy=False):
     finally:
         rmf.close()
 
-    # Remove any rows from the MATRIX and F_CHAN/N_CHAN where N_GRP is 0,
-    # since they do not add any data. This is to match the Crates backend.
+    return data, filename
+
+
+def get_rmf_data(arg, make_copy=False):
+    """arg is a filename or a HDUList object.
+
+    Notes
+    -----
+    The RMF format is described in [1]_.
```

Comment:
Can we avoid the use of `[1]` as a reference as this causes problems for sphinx in many situations? see https://github.com/sherpa/sherpa/pull/1813 for example of why that can trip sphinx and how to avoid it.

Reply:
At the moment we don't actually include the backend modules in the sphinx documentation. It's something we could think about doing, but

a) I'd rather not do that here
b) is complicated

There's things to be said about the breakdown I made over what modules are included in the sphinx docs, and where they are documented, but that's also for a different time and place!